cirq/work/observable_grouping_test.py

```diff
@@ -0,0 +1,54 @@
+# Copyright 2020 The Cirq developers
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import cirq
+
+
+def test_group_settings_greedy():
+    qubits = cirq.LineQubit.range(4)
+    q0, q1, q2, q3 = qubits
+    terms = [
+        0.1711977489805745 * cirq.Z(q0), 0.17119774898057447 * cirq.Z(q1),
+        -0.2227859302428765 * cirq.Z(q2), -0.22278593024287646 * cirq.Z(q3),
+        0.16862219157249939 * cirq.Z(q0) * cirq.Z(q1),
+        0.04532220205777764 * cirq.Y(q0) * cirq.X(q1) * cirq.X(q2) * cirq.Y(q3),
+        -0.0453222020577776 * cirq.Y(q0) * cirq.Y(q1) * cirq.X(q2) * cirq.X(q3),
+        -0.0453222020577776 * cirq.X(q0) * cirq.X(q1) * cirq.Y(q2) * cirq.Y(q3),
+        0.04532220205777764 * cirq.X(q0) * cirq.Y(q1) * cirq.Y(q2) * cirq.X(q3),
+        0.12054482203290037 * cirq.Z(q0) * cirq.Z(q2), 0.16586702409067802 *
+        cirq.Z(q0) * cirq.Z(q3), 0.16586702409067802 * cirq.Z(q1) * cirq.Z(q2),
+        0.12054482203290037 * cirq.Z(q1) * cirq.Z(q3),
```

Comment:
there is too much happening in this test - do we really need 8 elements for the ZZZZ group? Why are these particular coeffs interesting?

Reply:
this is the hydrogen hamiltonian from the openfermion example

Reply:
changed test name to clarify